src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -178,6 +198,7 @@ export class CodeSuggestionWebviewProvider
       ['arrow-right-dark', 'svg'],
       ['arrow-left-light', 'svg'],
       ['arrow-right-light', 'svg'],
+      ['learn-icon-dark', 'svg'],
```

Comment:
Is the `light` missing for light themes? Does theme change work fine?

Reply:
I tested the views in dark, light and high contrast - all nicely visible. This dark is only a bit darker than the light one, both are purple. There are two different colours because the icons are different shades of purple between CODE & OSS.

Reply:
Ok, will change CODE icon colour to match the current OSS icon colour